src/workerd/api/node/tests/async_hooks-nodejs-test.wd-test

```diff
@@ -0,0 +1,15 @@
+using Workerd = import "/workerd/workerd.capnp";
```

Comment:
As a todo for later... I wonder if we can simplify adding these tests by having bazel auto-generate the wd-test files for us based on wd_test rule in the bazel. We could easily extend that to include compat flag, compat flags, etc. Would be nice not to have to add multiple files every time we add a single test.

Reply:
+1